pages/[slug].js

```
@@ -1,399 +1,399 @@
+          return `<img src="${emojiURL}" alt="${emojiName}" style="height: 1em; vertical-align: text-bottom;" />`
+        })
+      )
+    }
+    replaceEmoji()
+  }, [])
+
+  return (
+    <Text
+      as={BaseStyles}
+      sx={{ my: [2, 4], fontSize: [2, 3] }}
+      dangerouslySetInnerHTML={{ __html: html }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)